doc/source/index.rst

```diff
@@ -40,6 +40,8 @@ To install the development version of this package::
    acszpt
    acsphotcte
    satdet
+   findsat_mrt
+   utils_findsat_mrt
```

Comment:
Why can't MRT only have one page?

Reply:
It can. I modified the rst files. utils_findsat_mrt.rst is no longer needed and has been deleted